packages/apiserver/src/resources/statuses/status.entity.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from 'typeorm';
 import { Material } from '../materials/material.entity';
 import { Person } from '../persons/person.entity';
+import { StatusTypes } from '../../../../global-types/src';
```

Comment:
`@mimir/global-types`?

Reply:
it doesn't work, so for some reason typeorm needs an absolute path to the file